nautobot_floor_plan/templates/nautobot_floor_plan/inc/floorplan_svg.html

```diff
@@ -1,8 +1,27 @@
 {% load buttons %}
 
 {% if floor_plan %}
-    <h4>{{ floor_plan }} {% edit_button floor_plan %}</h4>
-    Use shift+scroll wheel to zoom in or out. Click and drag to scroll.
+    <h4>{{ floor_plan }}</h4>
+    <div class="row">
+        <div class="col-md-1"></div>
+        <div class="col-md-8">
+            Use shift+scroll wheel to zoom in or out.
+            Click and drag to scroll.<br>
+            Add a tile by clicking the plus symbol on the grid OR by using the "Add Tile" button.
+        </div>
+        <div class="col-md-2">
+            <div class="btn-group btn-group-sm btn-group-justified" role="group">
+                <a href="{% url 'plugins:nautobot_floor_plan:floorplantile_add' %}?floor_plan={{ floor_plan.pk }}" class="btn btn-info btn-sm " id="add-floor-plan">
```

Comment:
I'd vote for `btn-primary` instead of `btn-info` to be more consistent with other "Add" buttons in the UI.